charts/ia-case-api/values.preview.template.yaml

```diff
@@ -81,7 +81,7 @@ ia-case-payments-api:
       CCD_URL: http://{{ .Release.Name }}-ccd-data-store-api
 
 ia-case-documents-api:
-  enabled: false #set to true if testing against a local branch
+  enabled: true #set to true if testing against a local branch
```

Comment:
If the comment is correct, I presume this needs reverting?

Otherwise get rid of the comment (or update it accordingly).